src/utils/SpokeUtils.ts

```diff
@@ -380,3 +386,8 @@ export async function findFillBlock(
 
   return lowBlockNumber;
 }
+
+// Determines if the input address (either a bytes32 or bytes20) is the zero address.
+export function isZeroAddress(address: string): boolean {
+  return address === ZERO_ADDRESS || address === ZERO_BYTES;
+}
```

Comment:
@bmzig This isn't related to the depositId changes; just want to make sure it's a conscious decision to have it included in this PR.

Reply:
I kept this since other PRs point to this one, and this is used in those PRs.